### PR TITLE
Pulled out FASTA/Q writing to single function

### PR DESCRIPTION
**Pull request by jessicamizzi**
Added a single function, write_record, for FASTA/Q writing, as described in issue #328

### CODE REVIEW

**Comment by jessicamizzi** [comment]
- [ ] Is it mergable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage.
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by jessicamizzi** [comment]
I noticed the coverage was not complete, so I ran diff-cover and all of my changes are completely covered by the tests. Can someone update the test coverage threshold? @mr-c @bocajnotnef @luizirber

**Comment by bocajnotnef** [comment]
retest this please

**Comment by bocajnotnef** [comment]
It looks like Jenkins is marking your build unstable because you introduced pep8 errors.

**Comment by bocajnotnef** [comment]
retest this please

**Comment by jessicamizzi** [comment]
Ready for review and merge @bocajnotnef @wrightmhw @b-wyss @brtaylor92

**Comment by ctb** [comment]
I'd like to take a look at it, too.  Over the weekend probably :)

On Fri, Jan 23, 2015 at 11:36:01AM -0800, jessicamizzi wrote:
> Ready for review and merge @bocajnotnef @wrightmhw @b-wyss @brtaylor92 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ged-lab/khmer/pull/736#issuecomment-71251540
-- 
C. Titus Brown, ctb@msu.edu

File: khmer/utils.py [hunk 1]
```diff
@@ -18,4 +18,19 @@ def print_error(msg):
     print >>sys.stderr, msg
 
 
+def write_record(record, fp):
+    """
+        Writes output sequence and returns str.
```
Comment (ctb) [inline khmer/utils.py]: spacing is off for line 23 - maybe a tab instead of spaces?

Comment (ctb) [inline khmer/utils.py]: Oh, I see, you're taking your cue from print_error -- please fix both to have the docstring start at the same indentation as """ ;)

**Comment by ctb** [comment]
LGTM; fix the indentation, make it mergeable, and we'll be good!

Please reference the original issue somewhere - perhaps in the pull request description, up top (which you can just edit).

Very nice cleanup, BTW. This is how to do it!

**Comment by luizirber** [comment]
:clap: :+1:

**Pull request merged**